packages/webapp/src/utils/developer-constants.js

```diff
@@ -1,44 +1,46 @@
-const DEVELOPER_CHECK_UPLOAD_METRIC = 'developer/check-metric-file'
-const DEVELOPER_CONFIRM_DEV_DETAILS = 'developer/confirm-development-details'
-const DEVELOPER_DOWNLOAD_METRIC_FILE = 'developer/download-metric-file'
-const DEVELOPER_TASKLIST = 'developer/record-gains-task-list'
-const DEVELOPER_ROUTING_REGISTER = 'developer/routing-register'
-const DEVELOPER_ROUTING_RESULT = 'developer/routing-result'
-const DEVELOPER_ROUTING_SOLD = 'developer/routing-sold'
-const DEVELOPER_BNG_NUMBER = 'developer/biodiversity-gain-site-number'
-const DEVELOPER_CHECK_ANSWERS = 'developer/check-answers'
-const DEVELOPER_CONFIRM_OFF_SITE_GAIN = 'developer/confirm-off-site-gain'
-const DEVELOPER_CONSENT_AGREEMENT_UPLOAD = 'developer/consent-agreement-upload'
-const DEVELOPER_AGREEMENT_CHECK = 'developer/consent-agreement-check'
-const DEVELOPER_DOWNLOAD_CONSENT_FILE = 'developer/download-consent-file'
-const DEVELOPER_DEVELOPMENT_PROJECTS = 'developer/development-projects'
-const DEVELOPER_CONTINUE_DEVELOPMENT_PROJECT = 'developer/continue-development-project'
-const DEVELOPER_NEW_DEVELOPMENT_PROJECT = 'developer/new-development-project'
-const DEVELOPER_NEED_METRIC = 'developer/need-metric'
-const DEVELOPER_AGENT_ACTING_FOR_CLIENT = 'developer/agent-acting-for-client'
-const DEVELOPER_IS_AGENT = 'developer-is-agent'
-const DEVELOPER_CHECK_DEFRA_ACCOUNT_DETAILS = 'developer/check-defra-account-details'
-const DEVELOPER_LANDOWNER_OR_LEASEHOLDER = 'developer/landowner-or-leaseholder'
-const DEVELOPER_UPLOAD_CONSENT_TO_USE_GAIN_SITE = 'developer/upload-consent-to-use-gain-site'
-const DEVELOPER_APPLICATION_BY_INDIVIDUAL_OR_ORGANISATION = 'developer/applying-individual-organisation'
-const DEVELOPER_CLIENT_INDIVIDUAL_ORGANISATION = 'developer/client-individual-organisation'
-const DEVELOPER_CLIENTS_NAME = 'developer/clients-name'
-const DEVELOPER_CLIENTS_ORGANISATION_NAME = 'developer/clients-organisation-name'
-const DEVELOPER_NEED_PROOF_OF_PERMISSION = 'developer/need-proof-of-permission'
-const DEVELOPER_UPLOAD_WRITTEN_AUTHORISATION = 'developer/upload-written-authorisation'
-const DEVELOPER_CHECK_WRITTEN_AUTHORISATION_FILE = 'developer/check-written-authorisation-file'
-const DEVELOPER_CHECK_PLANNING_DECISION_NOTICE_FILE = 'developer/check-planning-decision-notice-file'
-const DEVELOPER_UPLOAD_PLANNING_DECISION_NOTICE = 'developer/upload-planning-decision-notice'
-const DEVELOPER_DOWNLOAD_WRITTEN_AUTHORISATION = 'developer/download-written-authorisation-file'
-const DEVELOPER_DOWNLOAD_PLANNING_DECISION_FILE = 'developer/download-planning-decision-file'
-const DEVELOPER_DEFRA_ACCOUNT_NOT_LINKED = 'developer/defra-account-not-linked'
-const DEVELOPER_UPLOAD_CONSENT_TO_ALLOCATE_GAINS = 'developer/upload-consent-to-allocate-gains'
-const DEVELOPER_CHECK_CONSENT_TO_USE_GAIN_SITE_FILE = 'developer/check-consent-file'
-const DEVELOPER_PLANNING_DECISION_UPLOAD = 'developer/planning-decision-upload'
+const routes = {
```

Comment:
Revert to the old way of doing route constants

Allocation and Registration differ in the way constants are used than Credits Purchase (for historical reasons), so we should keep Allocation consistent and follow its conventions for this PR

Reply:
done